dropboxfs.py

```diff
@@ -169,6 +169,7 @@ def read(self, amt=None):
             return self.r.read(amt)
         else:
             self.close()
+            return ''
```

Comment:
@giampaolo Python 3 is not currently supported. There are other issues that would need to be addressed as well to get it working.